app/views/home/docs.html.erb

```diff
@@ -77,7 +77,7 @@
     <a class="ext" target="_blank" href='http://www.loc.gov/standards/sru/sru-2-0.html'>OASIS Search Retrieve via URL
       2.0 specification</a>.  When requested, facet information is added to the end of the ATOM feed in order to maintain
       the ATOM response compatibility.
-    The OSDD request requires a clientId query parameter used to uniquely identify the client as illustrated in the
+    While not required, the OSDD request allows for a clientId query parameter used to uniquely identify the client as illustrated in the
```

Comment:
Put the word 'recommended' somewhere in here.

Reply:
Reworked this sentence